src/symmetry.jl

```diff
@@ -93,18 +108,16 @@ function PermutationGroups.order(el::KleinPermElement)
 end
 
 struct KleinPermGroup <: Group end
-Base.one(::Union{KleinPermGroup, KleinPermElement}) = KleinPermElement(Perm(3), one(KleinGroup()))
+Base.one(::Union{KleinPermGroup, KleinPermElement}) = KleinPermElement(one(G3), one(KleinGroup()))
 # See https://github.com/blegat/CondensedMatterSOS.jl/pull/31#issuecomment-1717665659
-symmetric_group(n) = PermutationGroups.PermGroup(PermutationGroups.Perm([2,1]), PermutationGroups.Perm([2:n;1]))
-function PermutationGroups.gens(::KleinPermGroup)
-    els = [KleinPermElement(Perm(3), KleinElement(1))]
-    for g in gens(symmetric_group(3))
+function PermutationGroups.gens(g::KleinPermGroup)
+    els = [one(g)]
+    for g in gens(G3)
```

Comment:
@blegat this is where the bug was introduced;
The old version generated `KleinPermGroup`, since `Sym(3)` acts transively on non-trivial `KleinElement`s, the new one doesn't since `KleinElement(0)` is fixed under `Sym(3)`.

I just updated the function without thinking ;)